package.json

```
@@ -1,8 +1,8 @@
 {
-  "name": "ts-template",
+  "name": "ubiquibot-config",
```

```suggestion
  "name": "@ubiquibot/configuration",
```

That name is mixing up with our runtime `.yml` configs